ksqldb-udf/src/test/java/io/confluent/ksql/schema/ksql/types/SqlArrayTest.java

```diff
@@ -36,7 +36,7 @@ public void shouldImplementHashCodeAndEqualsProperly() {
     new EqualsTester()
         .addEqualityGroup(SqlArray.of(SOME_TYPE), SqlArray.of(SOME_TYPE))
         .addEqualityGroup(SqlArray.of(SqlPrimitiveType.of(SqlBaseType.BOOLEAN)))
-        .addEqualityGroup(SqlMap.of(SqlPrimitiveType.of(SqlBaseType.BOOLEAN)))
+        .addEqualityGroup(SqlMap.of(SqlTypes.STRING, SqlTypes.BOOLEAN))
```

Comment:
This looks like it should be `SqlMap.of(SqlTypes.STRING, SqlPrimitiveType.of(SqlTypes.BOOLEAN))` instead?

Reply:
No difference. Same thing.